### PR TITLE
Use webclient ivewer in the Default UI

### LINKED ISSUE #82: Use webclient viewer for Default UI
See https://forum.image.sc/t/setting-up-omero-based-virtual-microscopy-from-mirax/49469/3

We use `/webclient/img_detail/ID` for the Categories UI, and this will use eg. iviewer if configured.
But for the older Default UI, we still use `/webgateway/img_detail/ID` which always shows the old webgateway viewer.
Need to update that.

### PR DESCRIPTION
**Pull request by will-moore**
The supports configuration to use e.g. iviewer as the default viewer

Fixes #82 .

To test, check that you get `webclient/img_detail/ID` (will be iviewer) for these viewers:
 - In the default gallery UI, browse Group -> Project and click thumbnail to open viewer
 - Click `Load all Images` to load all the thumbnails for a Dataset. Click on these to open viewer
 - Click on the Dataset Name to browse to the Dataset page. Click on thumbnail to open viewer

### CODE REVIEW

**Comment by jburel** [comment]
Screenshot 2021-03-16 at 13 45 42


If I get that correctly, the image will be opened in the default viewer, whatever is set. In our configuration, it is iviewer.

File: omero_gallery/templates/webgallery/dataset_thumbs.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% for i in dataset.listChildren %}
-    <a href="{% url 'webgateway_full_viewer' i.id %}">
+    <a href="{% url 'web_image_viewer' i.id %}">
         <img title="{{ i.getName }}" src="{% url 'webgateway_render_thumbnail' i.id 96 %}" />
     </a>
 {% endfor %}
```
Comment (jburel) [inline omero_gallery/templates/webgallery/dataset_thumbs.html:5]: could add a line at  the end of the file

Comment (will-moore) [reply omero_gallery/templates/webgallery/dataset_thumbs.html]: Done

**Comment by jburel** [comment]
I have changed the configuration of omero-web, the old viewer is now the default.
Clicking on a thumbnail, the old viewer is used as expected
Screenshot 2021-03-24 at 10 09 10

**Comment by jburel** [comment]
This PR will use whatever is set as the default viewer, this offers more flexibility and it is what a user will expect

**Comment by sbesson** [comment]
There have been quite a few PRs merged recently including this one addressing some user feedback. Is it worth making a quick patch release of `omero-gallery` (that would could consume immediately in the upcoming `prod96` at the IDR level)?

**Comment by jburel** [comment]
I was going to propose a patch release too.
@will-moore what do you think?

**Comment by will-moore** [comment]
Yes, it would be great to get these new features/fixes released,  thanks.

**Comment by jburel** [comment]
Ok I will prep a release

**Pull request merged**